app/helpers/layout_helper.rb

```diff
@@ -1,8 +1,4 @@
 module LayoutHelper
-  def mount_layout
-    mount_react_component('Layout', "#layout", layout_data.to_json)
```

Comment:
can we avoid this change so the PR in katello won't be required?

Reply:
removed `react_app_helper`